pkg/cli/quit.go

```diff
@@ -270,24 +267,3 @@ func getAdminClient(ctx context.Context, cfg server.Config) (serverpb.AdminClien
 	}
 	return serverpb.NewAdminClient(conn), finish, nil
 }
-
-// grpcConfusedErrConnClosedByPeer returns true if the given error
```

Comment:
If this were still needed, how would we find out?

Reply:
TestDockerCLI will start to flake again.